Common/Tasks/checkDataModelMC.cxx

```diff
@@ -0,0 +1,68 @@
+// Copyright 2019-2020 CERN and copyright holders of ALICE O2.
+// See https://alice-o2.web.cern.ch/copyright for details of the copyright holders.
+// All rights not expressly granted are reserved.
+//
+// This software is distributed under the terms of the GNU General Public
+// License v3 (GPL Version 3), copied verbatim in the file "COPYING".
+//
+// In applying this license CERN does not waive the privileges and immunities
+// granted to it by virtue of its status as an Intergovernmental Organization
+// or submit itself to any jurisdiction.
+///
+/// \brief Load all tables in the data model to check if they can be read correctly
+/// \author
+/// \since
+
+#include "Framework/runDataProcessing.h"
+#include "Framework/AnalysisTask.h"
+
+using namespace o2;
+using namespace o2::framework;
+using namespace o2::framework::expressions;
+
+/// Check the complete list of indices of final-state daughters of an MC particle.
+/// \param particlesMC  table with MC particles
+/// \param particle  MC particle
+template <typename T>
+void checkDaughters(const T& particlesMC,
+                    const typename T::iterator& particle)
+{
+  if (particle.has_daughters()) {
```

Comment:
In principle this line can be skipped because in that case the list in the next line is simply empty, but this can be changed later.